src/layouts/GridView.jsx

```diff
@@ -20,11 +23,76 @@ const useStyles = makeStyles(() => ({
 
 export default function GridView() {
   const classes = useStyles();
-  const { cardData } = useGridData(currentConfig.contentData);
+  const { category = "home" } = useParams();
+  const [cardData, setCardData] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  const fetchData = () => {
+    axiosPrivate
+      .get(`/api/${currentConfig.contentData}`, {
+        params: {
+          content_category: category,
+          lang: "english"
```

Comment:
the lang's value is hardcoded , did you do it  for time being , it could be  set from  GlobalSetting or  personal setting  stored in local storage ?

Reply:
Yes right now its static but after that implementation we will use it from there